GOVERNANCE.md

```diff
@@ -0,0 +1,122 @@
+# Governance
+
+## Glossary
+
+* **Advisory Committee.**  A group of people with representation from a variety of AMP's constituencies including Users, End-users and Collaborators who provide advice to the Technical Steering Committee.
+
+* **Collaborators.**  People who have been granted write-access to ampproject repositories.
+
+* **Contributors**.  People who have opened a pull request or an issue, or commented on an issue. Contributors are granted certain permissions on the repositories to make it easier to participate--e.g. assigning issues, attaching labels to issues, etc.
+
+* **Users.** Developers who use AMP but haven’t contributed to it (yet).
+
+* **End-users**. People who consume content distributed using the AMP format.
+
+* **Owners**.  People who are most familiar with a particular set of code and who have been granted the power (by Owners at a higher level) for approving changes to that code.  AMP will use a system similar to [Chromium's OWNERS](https://chromium.googlesource.com/chromium/src/+/master/docs/code_reviews.md#OWNERS-files) system.  The AMP Project will follow Chromium's [expectation for owners](https://chromium.googlesource.com/chromium/src/+/master/docs/code_reviews.md#expectations-of-owners).
```

Comment:
"Similar to" seems a little vague for a governance policy

Reply:
We're working on the actual software. @mrjoro Maybe we can at least say when we can define this more definitely?

Reply:
@rsimha and @erwinmombay are working on getting the details of how this will work published soon (well before the governance review period is done).  We'll add a link here when it's done.

Arguably we could just leave this line out of the governance proposal, since it's not really about the governance exactly, and we can leave it to the discretion of the TSC how to actually enforce this.

We should also fix the parenthetical to be "(by the TSC or Owners at a higher level)" since the TSC is the one that provides this power.

Reply:
There may also be different policies in place between repositories. The owner process makes sense for `amphtml` but other lower traffic repositories may opt for something simpler.

Reply:
Agree with @mrjoro that the specifics of how Owners are implemented should be left to the discretion of the TSC. It's still useful to have the high-level concept detailed in the same document as Contributors and Collaborator so that the progress ladder is easily discoverable and understandable.

Might be worth prefixing it with "In certain repositories…" or the like, to adress @cramforce's point.

Reply:
And while we're at it, we should probably fold in and adapt the requirements listed on the Chromium website and reference that page informationally only (i.e. in a non-binding way).